style/base.css

```diff
@@ -8,31 +8,32 @@
     opensarlab-diskspace
 */
 .diskspace {
-    position: relative;
-    display: inline-block;
-    /*border-bottom: 1px dotted black;*/
+  position: relative;
+  display: inline-block;
+
+  /* border-bottom: 1px dotted black; */
 }
 
 .diskspace .red {
-    background-color: rgba(255, 200, 200, 0.75);
-    padding: 5px;
-    border-radius: 25px;
+  background-color: rgba(255 200 200 / 75%);
```

Comment:
Apparently the linter converts the CSS rgb values to the more modern format. https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb#formal_syntax Who knew?